profiles/templates/profiles/profile.html

```diff
@@ -3,7 +3,40 @@
 
 {% block content %}
 
-    <!-- TO DO : build profile page -->
-    TEST PROFILE PAGE
+<!DOCTYPE html>
+    <h1>Profile Page</h1>
+    <h2>You are logged in as:</h2>
 
+<!-- Profile display -->  
+ 
+    <div class="container-fluid">
+        <div class="row">
+
+            <div class="col-md-4">
+                <h4>Username :</h4>

```

Comment:
@aticodein 
Would you mind fixing some of the extra spaces on the code?
`<h4>Username:</h4>`

Reply:
Changed that